spec/unit/mutant/ast/pattern/source_spec.rb

```diff
@@ -29,12 +29,11 @@ def apply
     end
 
     context 'on non unix newline' do
-      let(:input) { "a\r\nb" }
-
+      let(:string) { "a\r\nb" }
       let(:line_index) { 0 }
 
       it 'considers the \r as part of the previous line' do
-        expect(apply).to eql('a')
+        expect(apply).to eql("a\r")
```

Comment:
@mbj I assume this is the behavior you meant by the docstring but maybe this deserves a closer look?